### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by DonIsaac**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (tybug) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: these need to be headers (small, h3 maybe?) or there needs to be two linebreaks afterwards, one linebreak puts them on the same line 
image

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (tybug) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: I'd clarify these are example reproductions, theirs might not be the same

**Review by tybug — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (DonIsaac) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: yeah this basically the default template, and it assumes the project is web based. Not entirely sure what *exactly* we should have in these, got any ideas?

Comment (tybug) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: I can't come up with anything good...I like the rest of the template (even if it is generic) but these steps are too much IMO. Maybe just remove the steps and say "List the steps to reproduce to the behavior."

File: .github/ISSUE_TEMPLATE/endpoint_request.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+---
+
+### Basic Info
+
+- **Is this for a new endpoint, a new collection, or an update to an existing endpoint?**
+- **Is this a breaking change?** 
+- **What endpoint(s) are you looking for?** What new endpoint(s) do you want? Or,
+if this is for an existing endpoint, which one? If you are not sure what endpoints
+apply, describe the type of data being served below.
```
Comment (tybug) [inline .github/ISSUE_TEMPLATE/endpoint_request.md:16]:
```suggestion
apply, describe what type of data you would like umdio to provide below.
```

**Review by tybug — CHANGES_REQUESTED** [review]
lgtm other than this, definitely an improvement over the defaults

File: .github/ISSUE_TEMPLATE/endpoint_request.md (hunk 3 above)
Comment (tybug) [reply .github/ISSUE_TEMPLATE/endpoint_request.md:16]: or stick with `the` instead of `what` if you prefer, I don't really care on second thought

**Comment by DonIsaac** [comment]
> lgtm other than this, definitely an improvement over the defaults

what does lgtm mean?

**Comment by tybug** [comment]
looks good to me (I'll tell you honestly I only started using it because I've seen others using it on PR approvals, and I think that's just about the only time anyone ever uses lgtm)

**Review by tybug — APPROVED** [review]

**Pull request merged**